website/docs/components/table/table/partials/code/how-to-use.md

```diff
@@ -122,7 +122,7 @@ For clarity, there are a couple of important points to note here:
 
 !!! Info
 
-This component takes advantage of the `sort-by` helper provided by [ember-composable-helpers](https://github.com/DockYard/ember-composable-helpers).
+This component takes advantage of the `sort-by` helper provided by [@nullvoxpopuli/ember-composable-helpers](https://github.com/NullVoxPopuli/ember-composable-helpers).
```

Comment:
we introduced a new `AdvancedTable` component with a similar copy – would you be able to update those links as well?